### PR TITLE
Added pyblake2

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Closes https://github.com/conda-forge/staged-recipes/issues/4499

cc @jakirkham

### CODE REVIEW

File: recipes/pyblake2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+  number: 0
+  script:
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python  # [win]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: This `requirements` section appears to be a duplicate and can be dropped.

File: recipes/pyblake2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - msinttypes  # [win]
+    - vc 9        # [win and py27]
+    - vc 10       # [win and py34]
+    - vc 14       # [win and py>=35]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Given that we depend on `python`, I don't think we need the `vc` dependency as well.

File: recipes/pyblake2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - pip
+  run:
+    - python
+    - vc 9    # [win and py27]
+    - vc 10   # [win and py34]
+    - vc 14   # [win and py>=35]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Same as above.

File: recipes/pyblake2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+build:
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Feel free to keep or drop the `py34` as you like.

File: recipes/pyblake2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+requirements:
+  build:
+    - python  # [win]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - msinttypes  # [win]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Don't think this should be needed for `py>=35` as the system should provide it then. So maybe can add `py<35`.

File: recipes/pyblake2/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    - python
+    - vc 9    # [win and py27]
+    - vc 10   # [win and py34]
+    - vc 14   # [win and py>=35]
+
+test:
+  imports:
+    - pyblake2
+  commands:
+    - conda inspect linkages {{ name|lower }}  # [linux]
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Usually we make the selector here `unix`. It works fine on macOS and Linux. Also provides us some info that `conda inspect objects` does not.

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk. Looks pretty good. Made some minor suggestions above.

File: recipes/pyblake2/meta.yaml (hunk 4 above)
Comment (jakirkham) [reply recipes/pyblake2/meta.yaml]: Ah sorry. This was confusing. We should drop the `features` altogether as this is `python` and so already tracks `vc` that way.

**Comment by jakirkham** [comment]
Once `features` are dropped should be good to go.

**Comment by mariusvniekerk** [comment]
@jakirkham ready

File: recipes/pyblake2/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+
+about:
+  home: https://github.com/dchest/pyblake2
+  license_family: MIT
+  license: MIT
+  license_file: COPYING
+  summary: |
+    pyblake2 is an extension module for Python implementing BLAKE2 hash function.
+    BLAKE2 is a cryptographic hash function, which offers highest security while being as fast as MD5 or SHA-1,
```
Comment (jakirkham) [inline recipes/pyblake2/meta.yaml]: Probably should end line with a `.` instead of `,`. That said, this is a really minor thing and not blocking.

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk.

**Pull request merged**